src/templates/new_relations.tsv

```diff
@@ -1,2 +1,16 @@
 relation id (required)	type	label (required)	definition (required)	parent relation	creator ORCID (required)	Date created (required)	relation characteristics	domain	range	inverse property	example of use	synonyms	curator/usage notes
 ID	TYPE	AL rdfs:label@en	A IAO:0000115	SP %	AI dc:contributor	AT dc:created^^xsd:dateTime	CHARACTERISTIC SPLIT=|	DOMAIN	RANGE	IP %	A rdfs:comment	A oboInOwl:hasExactSynonym SPLIT=|	A IAO:0000116
+20001 TYPE  isAnatomicalEntityFor A relation describing an anatomical featureof the referenced object with a material entity   AI dc:contributor0000-0002-6571-0033 AT dc:created^^xsd:16.03.2022 

```

Comment:
You need to add `RO:` prefix for all IDs, and pad the ID to have exactly 7 digits, like:

```
RO:0020001
```